DEV-3/NumberConverter/NumberHandler.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections;
+using System.Text;
+
+namespace NumberConverter
+{
+  class NumberHandler
+  {
+    private int systemBase;
+
+    public int Number { get; set; }
+
+    public int SystemBase
+    {
+      get
+      {
+        return systemBase;
+      }
+      set
+      {
+        if (value < 2 || value > 20)
+        {
+          throw new ArgumentException();
+        }
+        systemBase = value;
+      }
+    }
+
+    public NumberHandler(int number, int systemBase)
+    {
+      Number = number;
+      SystemBase = systemBase;
+    }
+
+    /// <summary>
+    /// method converts the number to the required number system
+    /// </summary>
+    /// <returns></returns>
+    public string ConvertNumber()
+    {
+      StringBuilder newNumber = new StringBuilder();
+      ArrayList arrayOfResidues = new ArrayList();
+
+      arrayOfResidues = CalculateRemainderOfDivision();
+      arrayOfResidues.Reverse();
+
+      if (systemBase > 10)
+      {
+        foreach (int el in arrayOfResidues)
+        {
```

Comment:
Better do it uniformly in ConvertNumberIntoLetter method both for el >= 10 and el < 10